charts/rancher-vsphere-csi/Chart.yaml

```
@@ -1,26 +1,24 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: vSphere CSI
-  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.28.0-0'
+  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.29.0-0'
   catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/os: linux,windows
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/rancher-version: '>= 2.8.0-0'
   catalog.cattle.io/release-name: vsphere-csi
 apiVersion: v1
-appVersion: 3.0.2-rancher2
+appVersion: 3.1.2-rancher2
```

`-rancherX` version is used to track Rancher (vs. upstream)-specific changes to this chart (e.g. see https://github.com/rancher/vsphere-charts/pull/61 when it got bumped). Since we're changing upstream I think we should reset it to `-rancher1`. It doesn't matter _that_ much but will be cleaner if we reset it.

```suggestion
appVersion: 3.1.2-rancher1
```